apps/auth/config.py

```
@@ -0,0 +1,30 @@
+if not os.getenv("REDIS_PORT"):
+    print("\033[33m[WARN]\033[0m REDIS_PORT not set, using default value of 6379")
+
+MONGO_URI = os.getenv("MONGO_URI")
+REDIS_HOST = os.getenv("REDIS_HOST")
+REDIS_PORT = os.getenv("REDIS_PORT", 6379)
+
+if not os.getenv("PASSWORD_MIN_LENGTH"):
+    print("\033[33m[WARN]\033[0m PASSWORD_MIN_LENGTH not set, using default value of 8")
+if not os.getenv("PASSWORD_MAX_LENGTH"):
+    print(
+        "\033[33m[WARN]\033[0m PASSWORD_MAX_LENGTH not set, using default value of 32"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)